datafusion/physical-plan/src/spill.rs

```diff
@@ -85,3 +85,104 @@ fn read_spill(sender: Sender<Result<RecordBatch>>, path: &Path) -> Result<()> {
     }
     Ok(())
 }
+
+/// Spill the `RecordBatch` to disk as smaller batches
+/// split by `batch_size_rows`
+/// Return `total_rows` what is spilled
+pub fn spill_record_batch_by_size(
```

Comment:
Where is this function used other than in test? I don't find it.

Reply:
Yes, Im planning to keep it and reuse it in row_hash in following PR, basically the subbatch slicing is from `row_hash.rs`